result/lacplagmember.yaml

```diff
@@ -0,0 +1,148 @@
+      type: object
+      properties:
+        lag_names:
+          description: >-
+            The names of LAG (ports group) for which LACP metrics to be returned.
+            An empty list will return metrics for all LAGs.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - '/components/schemas/Lag/properties/name'
+        lag_member_names:
+          description: >-
+            The names of LAG member (port) for which LACP metrics to be returned.
+            An empty list will return metrics for all LAG members.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - '/components/schemas/Port/properties/name'
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned.
+            The name of LAG and LAG member can not be excluded.
+          type: array
+          items:
+            type: string
+            enum:
+              - lacp_in_pkts
+              - lacp_out_pkts
+              - lacp_rx_errors
+              - activity
+              - timeout
+              - synchronization
+              - aggregatable
+              - collecting
+              - distributing
+              - system_id
+              - oper_key
+              - partner_key
+              - port_num
+              - partner_port_num
+
+    LacpLagMember.Metric:
+      description: >-
+        LACP metrics (statistics) per LAG member.
+      type: object
+      properties:
+        lag_name:
+          description: >-
+            The name of a LAG (ports group) configured with LACP.
+          type: string
+        lag_member_name:
+          description: >-
+            The name of a LAG member (port) configured with LACP.
+          type: string
+        lacp_in_pkts:
+          description: >-
+            Number of LACPDUs received.
+          type: integer
+          format: int64
+          minimum: 0
+        lacp_out_pkts:
+          description: >-
+            Number of LACPDUs transmitted.
+          type: integer
+          format: int64
+          minimum: 0
+        lacp_rx_errors:
+          description: >-
+            Number of LACPDUs receive packet errors.
+          type: integer
+          format: int64
+          minimum: 0
+        activity:
+          description: >-
+            Indicates participant is active or passive.
+          type: string
+          enum: [active, passive]
+        timeout:
+          description: >-
+            The timeout type (short or long) used by the participant.
+          type: string
+          enum: [short, long]
+        synchronization:
+          description: >-
+            Indicates whether the participant is in-sync or out-of-sync.
+          type: string
+          enum: [in_sync, out_sync]
```

Comment:
out_sync -> out_of_sync

Reply:
Based on OpenApi option flags. If we make it "out_of_sync", it can be odd for the user.
https://github.com/openconfig/public/blob/master/release/models/lacp/openconfig-lacp.yang

Reply:
Got it, OpenConfig calls it OUT_SYNC.  thanks.